### ISSUE TITLE
[@uifabric/example-app-base] Screen reader is not announcing any information about the code section which appears after activating "Show code" button

### ISSUE THREAD

**Issue by lucas-stanford** [microsoft/fluentui#16613]
**Test Environments:**
Version:  2004 (OS Build: 20236.1005)
Browser: Edge chromium 
Screen reader: Narrator, NVDA & JAWS
Bug id: 8771462

**Describe the issue:**
In the package @uifabric/example-app-base 

**Please provide a reproduction of the issue in a codepen:**
Unable to use codepen as the issue is located in @uifabric/example-app-base

**Actual Result:**
Code section appears after activating the "Show code" button but screen reader does not announce any information about the code section appearing 

**Expected Result:**
Screen reader should announce "Code of *component name* Hide code button" after activating "Show code" button

**Documentation describing expected behavior:**
N/A

![image](https://user-images.githubusercontent.com/3011776/105775169-3998c100-5f1b-11eb-8733-5cd7cb1dcf5a.png)

**Comment by gouttierre** [comment]
@lucas-stanford  - Thank you for the feedback into the issue you are experiencing.

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue with the citing of "Show code" button functionality?
cc @smhigley

**Comment by vanessgo** [comment]
@khmakoto Gentle Ping. Do you have an ETA on fixing this issue?
